relay-pii/src/processor.rs

```diff
@@ -200,23 +200,10 @@ impl<'a> Processor for PiiProcessor<'a> {
         replay.process_child_values(self, state)?;
         Ok(())
     }
-
-    fn process_event(
```

Comment:
Since removing scrub_graphl makes it match the default implementation we can simply remove this